styledes.css

```diff
@@ -0,0 +1,599 @@
+    display: flex;
+    flex-direction: row;
+    width: 175px;
+    align-items: flex-start;
+    text-align: center;
+    justify-content: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    font-size: 17px;
+    font-weight: 500;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    left: 12px;
+    margin: 8px;
+    cursor: pointer;
+    background-color: #fff;
+  }
+
+  .normalButnAbout:hover {
+    background: #6070ff;
+    color: #fff;
+  }
+
+  .normalButnAbout:active {
+    background: #2230d2;
+    color: #fff;
+  }
+
+  .skillList {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    position: absolute;
+    width: 43%;
+    height: 360px;
+    left: 614px;
+    top: 153px;
+    flex: 50%;
+    order: 0;
+    flex-grow: 0;
+    background: #000;
+  }
+
+  .dropdown_container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    width: 100%;
+    height: 216px;
+  }
+
+  #dropdown_header {
+    width: 100%;
+    height: 72px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .dropdown {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 12px;
+    left: 0;
+    gap: 12px;
+    width: 75%;
+    height: 120px;
+  }
+
+  #java_dropdown {
+    width: 122px;
+    height: 120px;
+    background: #f7f7f9;
+    border-radius: 8px;
+    flex: none;
+    order: 0;
+  }
+
+  .drp_txt {
+    position: relative;
+    width: 98px;
+    height: 24px;
+    left: 8px;
+    top: 50px;
+  }
+
+  .separate {
+    position: absolute;
+    height: 1;
+    left: 0;
+    right: 0;
+    bottom: 0;
+  }
+
+  .hr {
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+  }
+
+  .contactMe {
+    width: 100%;
+    height: 846px;
+    left: 0;
+    top: 4110px;
+    background-image: url(./asset/Contact/contactdesk.svg);
+    background-repeat: no-repeat;
+    background-size: 95% 95%;
+    background-position: 90% 80%;
+  }
+
+  .contactblock {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    align-content: center;
+    padding: 0;
+    gap: 20px;
+    position: absolute;
+    width: 57%;
+    height: 577px;
+    margin-top: 125px;
+    margin-left: 320px;
+  }
+
+  .contactTxt {
+    width: 100%;
+    height: 56px;
+    position: absolute;
+    margin-top: -55px;
+    text-align: center;
+    flex: 0;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .Contact_body_text {
+    align-content: center;
+    width: 100%;
+    height: auto;
+    left: 0;
+    top: 80px;
+    flex: 0;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .body-text {
+    margin-left: 105px;
+    justify-content: center;
+    align-content: center;
+    width: 70%;
+    top: 0;
+    height: fit-content;
+  }
+
+  .form {
+    display: flex;
+    flex-direction: column;
+    margin-top: 250px;
+    width: 100%;
+    height: auto;
+    margin-left: 24px;
+    margin-right: 24px;
+  }
```

Comment:
- [ ] Kindly ensure that the placeholder for each input is aligned correctly as shown in the Figma design.

### Your Design
![image](https://user-images.githubusercontent.com/68030297/221279717-28704726-99c4-4e79-99ab-88bc2a6be881.png)


### Figma Design
![image](https://user-images.githubusercontent.com/68030297/221279625-46cb63b4-4ace-4474-9556-0410565abb14.png)